src/components/icons/App.svelte

```diff
@@ -1,42 +1,26 @@
 <script>
-  let className = undefined;
-  export { className as class };
-  export let id = undefined;
-  export let tabindex = undefined;
-  export let focusable = false;
   export let title = undefined;
   export let style = undefined;
   export let dimension = 60;
 
-  $: ariaLabel = $$props["aria-label"];
-  $: ariaLabelledBy = $$props["aria-labelledby"];
-  $: labelled = ariaLabel || ariaLabelledBy || title;
+  let id = "icon-title-app";
   $: attributes = {
+    id,
```

Comment:
@mukhtyar it seems like the `id` is being applied to both the svg and title elements, which means it's not unique. More info here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/166/commits/9ff2407e77162347d964395260101aa905d46fd5